packages/sortable-behavior-bundle/README.md

```diff
@@ -32,6 +32,74 @@ return [
 ];
 ```
 
+## Usage
+
+### Basic usage
+```php
+
+namespace App\Admin;
+
+use Runroom\SortableBehaviorBundle\Admin\AbstractSortableAdmin;
+
+class ExampleAdmin extends AbstractSortableAdmin
+{
+    // you can use SortableAdminTrait instead AbstractSortableAdmin
```

Comment:
I guess it's better to recommend directly the usage of the trait.

Reply:
Done